### ISSUE TITLE
Dev: Document and request new API for Status widget from os.io backend team

### ISSUE THREAD

**Issue by dgutride** [fabric8-ui/fabric8-ux#503]
The status widget at the top of openshift.io will require a new API to provide all connected systems statuses (statii or statum?).   Once the design is complete, we should identify a potential json structure and the necessary data for the client to request.

**Comment by dgutride** [comment]
@joshuawilson  - we need to discuss this when you return...

**Comment by joshuawilson** [comment]
@dgutride there has been some work on this, we need to check with @pmuir

**Comment by dgutride** [comment]
No joint api will created - this issue is no longer relevant.
